src/content/docs/style-guide/images/embed-images.mdx

```diff
@@ -37,7 +37,7 @@ Here are some things to keep in mind when you're creating an image:
 * For screenshot captions, include the path in bold.
 * For video captions, include the approximate running time.
 * Do not link to an external image file. All images used in docs must be stored in [our GitHub repo](https://github.com/newrelic/docs-website).
-* Our default is light mode. If an SME provides you with screenshots in dark mode, please request new ones in light mode. If you are unable to get light mode images, then any screenshot is better than no screenshot at all. 
+* Our default is dark mode. If an SME provides you with screenshots in light mode, please request new ones in dark mode. If you are unable to get dark mode images, then any screenshot is better than no screenshot at all. 
```

Comment:
@nwhite222 thanks for making this update! Your changes look good! I have a few minor suggestions that are a bit out of the scope of this original request, but consider these minor changes:

* Spell out SME (subject matter expert).
* Add a blurb to our [image annotations](https://docs.newrelic.com/docs/style-guide/images/image-annotations/) page reminding us to confirm that any annotations we add are visible in dark and light modes.

Reply:
Sounds good! Thank you.